scripts/build

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
-set -e
-
-vite build
-cp _redirects build/_redirects
```

Comment:
Is this going to work? The script copied the `_redirects` file to the `build` folder, but you moved it to `public`.

Reply:
Yes the files that are in the `public` folder are copied to the `build` folder by `vite build`